wasm builds fail on get_root_path

Since eb124a30babf3b42ea9ad107653aee0570c678b6

```
error[E0599]: no function or associated item named `get_root_path` found for struct `WasmAssetIo` in the current scope
  --> /Users/robparrett/.cargo/git/checkouts/bevy_tiled-85a429c236b45725/d997e92/src/map.rs:61:44
   |
61 |         let root_dir = asset::WasmAssetIo::get_root_path();
   |                                            ^^^^^^^^^^^^^ function or associated item not found in `WasmAssetIo`
```

Things are generally working very well prior to this commit.

--- a/src/map.rs
+++ b/src/map.rs
@@ -4,7 +4,6 @@ use crate::{
 };
 use anyhow::Result;
 use bevy::{
-    asset,
     prelude::*,
     reflect::TypeUuid,
     utils::{HashMap, HashSet},
@@ -56,11 +55,9 @@ impl Map {
 
     pub fn try_from_bytes(asset_folder: &Path, asset_path: &Path, bytes: Vec<u8>) -> Result<Map> {
         #[cfg(all(not(target_arch = "wasm32"), not(target_os = "android")))]
-        let root_dir = asset::FileAssetIo::get_root_path();
-        #[cfg(target_arch = "wasm32")]
-        let root_dir = asset::WasmAssetIo::get_root_path();
-        #[cfg(target_os = "android")]
-        let root_dir = asset::AndroidAssetIo::get_root_path();
+        let root_dir = bevy::asset::FileAssetIo::get_root_path();
+        #[cfg(any(target_arch = "wasm32", target_os = "android"))]
+        let root_dir = PathBuf::from("");
         
         let map = tiled::parse_with_path(
             BufReader::new(bytes.as_slice()),

